src/components/cap-content-router.js

```diff
@@ -1,10 +1,24 @@
-import { html, LitElement } from "../lib/lit.js";
+import { html, css, LitElement } from "../lib/lit.js";
+
 import "./cap-volume.js";
 import "./cap-case.js";
 import "./cap-jurisdictions.js";
 import "./cap-reporter.js";
 
 export default class CapContentRouter extends LitElement {
+	static styles = [
+		css`
+			cap-case {
+				display: block;
+				flex: 0 0 58.33333%;
+				max-width: 58.33333%;
+				margin: auto;
+				padding-top: 2.5rem;
+				padding-bottom: 3rem;
+			}
+		`
+	];
+
```

Comment:
Would it feel reasonable to maintain all of CAP's original css names for pre-written HTML, but follow existing strategies for anything that isn't? 

I'm not sure if I explicitly mentioned this in any PR —  one of the CSS strategies cap static uses is [BEM](https://getbem.com/introduction/) (Block Element Modifier) so for example here, `cap-case` would be a block and it would be named `capCase`. 

I generally am not at all opposed to this work happening in another PR, if that feels like the easiest approach.

Reply:
Got it. Yes!! That seems extremely reasonable: as a matter of fact, I was assuming we'd want to do something like that.

I think let's follow up in a separate PR? I like the idea of this existing in the as-close-to-CAP version, and then being modified. Michael also suggested it might be easiest to see this land, and then we can figure out who can most efficiently refactor :-) As in, I'll throw something in Linear about refactoring it, and we can pick that off the queue.

Thanks for the link about BEM, and the examples. I've heard of it but haven't had occasion to work that way yet! Looking forward to getting up to speed with the technique and seeing how it plays out on a larger site!

Reply:
See ENG-573